src/components/data-exploration/generic-gene-table/SelectionActions.jsx

```diff
@@ -91,10 +93,13 @@ const SelectionActions = (props) => {
 
       {
         showCSV ? (
```

Comment:
I think we can remove this `showCSV` conditional. If you check the definition, the value of `showCSV` is ` onExportCSV !== null`, meanwhile `onExportCSV` is defined as `() => { setExportAlert(true); }`. Can you check if removing this will still render the results correctly?

If you decide to remove `showCSV`, don't forget to remove the definitions from the propTypes as well.